.template/addons/phrase_app/template.rb

```diff
@@ -0,0 +1,6 @@
+after_bundle do
```

Comment:
Do we need the `after_bundle` here as we are just copying and inserting into file?

Reply:
yeah I tried without after_bundle and there is some error relative to the `spec` as I remember